### ISSUE TITLE
Unable to get permissions for camera and microphone when using custom widget with own jitsi server.

### ISSUE THREAD

**Issue by erikderzweite** [vector-im/element-web#14915]
### Description

I'm trying to integrate a widget to my own jitsi server into a room. Creating a custom widget works, but I am unable to use camera and microphone, I get "Error obtainig microphone/camera permission" message. This is true for web client as well as desktop client. 

riot.im's Jitsi widget does not have the permissions problem and works. But if I add same jitsi.riot.im URL via custom widget then I cannot set permissions as well. 

### Steps to reproduce

- Add a custom widget with the URL to a custom jitsi conference room
- Try to activate microphone/camera

Logs being sent: no

### Version information

- **Platform**: web and desktop?

For the web app:

- **Browser**: Chromium 83.0.4103.61, Firefox 78.0.2
- **OS**: Ubuntu 18.04
- **URL**: riot-web 1.7.1

For the desktop app:

- **OS**: Ubuntu 18.04
- **Version**: 1.7.2

**Comment by jryans** [comment]
@turt2live Any guess what might be happening here?

**Comment by MaximumSU** [comment]
The same on Windows desktop/Chrome/Firefox.

**Comment by t3chguy** [comment]
Sounds like you denied the permissions directly on the jitsi_domain at some point in the past. Check your browser settings

**Comment by MaximumSU** [comment]
@t3chguy No. If i open the widget link directly in the browser's windows - no problem, all works, as expected.

**Comment by binex-dsk** [comment]
having the same problem on the desktop client with my jitsi meet instance

**Comment by erikderzweite** [comment]
As my client is self-hosted a workaround has been to switch to [Dimension](https://dimension.t2bot.io/) integration server. Works for Element-Desktop as well.

**Comment by ksadorf** [comment]
Hi, 

I face the same problem with another videoconference platform (https://whereby.com/). 
I don't know how to get the widget to access my microphone and camera.

Does someone have an idea to workaround this ?
